### ISSUE TITLE
First Person - Boarding a Ship Loses Camera Target

### ISSUE THREAD

**Issue by AdmiralRadish** [EloxZ/KerbalView#2]
When boarding a ship in First-Person mode, the camera can glitch out and lose all targets. When a boarding action occurs or the kerbal "disappears" the camera should return to the boarded vessel (ideally)

**Comment by elobdev** [comment]
Hello, I will copy what I posted in the forum:

I'm attaching the main camera to the kerbal, as a child, for two reasons: I didn't figure out yet how to copy the main camera with the exact configuration, If I create a new camera, it doesn't render planet surfaces.
And if I keep updating the camera with the kerbal position, it works well on land, but it can't keep his movement at high speeds/in orbit. This provokes that, when the game destroys the kerbal object (when boarding or dying), it destroys the main camera with him. I've tried to save the camera from destruction, but the code does unknown stuff when boarding a vessel and I guess it needs its original parent. So it does a weird thing: the ocean follows you to space. If I were able to create my own camera it would fix this issue.
